test/form/samples/supports-core-js/_expected.js

```diff
@@ -9133,7 +9133,7 @@ function requireEs_number_minSafeInteger () {
 	// `Number.MIN_SAFE_INTEGER` constant
 	// https://tc39.es/ecma262/#sec-number.min_safe_integer
 	$({ target: 'Number', stat: true, nonConfigurable: true, nonWritable: true }, {
-	  MIN_SAFE_INTEGER: -0x1FFFFFFFFFFFFF
+	  MIN_SAFE_INTEGER: -9007199254740991
```

Comment:
I am not completely sure about this one. I think this example shows that decimal numbers will never be longer than hexadecimal numbers and thus it is fine from a minifying perspective. People may complain from a readability perspective, but I believe that is not a big issue to be concerned about.
What I do not understand is why in line 9119, `MAX_SAFE_INTEGER` is NOT replaced. I would expect either neither or both are replaced.
But maybe it is called from a try-catch block or similar.

Reply:
Ah no, that is of course because MAX_SAFE_INTEGER is only a literal and not a UnaryExpression 🤦‍♂️. We could of course extend the logic to rendering literals themselves, but not necessarily in this PR.